src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs

```diff
@@ -236,6 +235,26 @@ public void VerifyAspireTemplate()
         cleanup.Execute(_sdkHelper, _scenarioTestInput.TestRoot, "aspire");
     }
 
+    [Fact]
+    [Trait("Category", "Workload")]
+    [Trait("SkipIfPlatform", "LINUX")]
+    [Trait("SkipIfPlatform", "OSX")]
```

Comment:
I think these maui tests should pass on Mac?

Reply:
It is likely a fixable machine problem, but installing xCode isn't probably something we should be doing as part of this execution I think?

Reply:
That's sorta surprising...It should be installed by default (it supports the build tooling).